synapse/storage/databases/main/__init__.py

```diff
@@ -184,6 +185,7 @@ async def get_users_paginate(
             name: search for local part of user_id or display name
             guests: whether to in include guest users
             deactivated: whether to include deactivated users
+            admins: whether to query only admins. admins are excluded if false. admins are included if absent.
```

Comment:
Can you update this to match the suggested phrasing in the docs?

Reply:
Comment adjusted.